src/app/layout.tsx

```diff
@@ -1,35 +1,40 @@
-import type { Metadata } from "next";
-import Logo from '../assets/logo.svg'
-import "./globals.css";
-import Image from "next/image";
+import type { Metadata } from "next"
+import Logo from "../assets/logo.svg"
+import "./globals.css"
+import Image from "next/image"
+import TeamImage from "../assets/team/team.png"
 
 export const metadata: Metadata = {
   title: "Coding bunker",
-  icons: "/favicon.svg"
-  
-};
+  icons: "/favicon.svg",
+  openGraph: {
+    title: "Coding bunker",
+    description:
+      "Unisciti a Coding Bunker, la community inclusiva per appassionati di programmazione e tecnologia. Scopri eventi, supporto per principianti ed esperti e proponi il tuo talk!",
+    images: TeamImage.src,
+    type: "website",
+    locale: "it_IT",
+    url: "https://codingbunker.it",
```

Comment:
@KristianLentino99 It is normal to have only the domain.com and not www. here?

Reply:
You're right, the URL here should have the `www`, fixed in 857a38ddef034dcb41ac0068406b92dc056390cb